tests/e2e_logging.py

```diff
@@ -464,13 +464,13 @@ def get_all_tx_ids():
 
 
 @reqs.description("Build a list of Tx IDs, check they transition states as expected")
-@reqs.supports_methods("log/private", "tx")
+@reqs.supports_methods("log/private", "node/tx")
```

Comment:
```suggestion
@reqs.supports_methods("app/log/private", "node/tx")
```

Reply:
I think should be node/tx -> tx instead, but I'll double check.

Reply:
Yes, that's only looking methods up under the app frontend. It needs to either stay unprefixed or be extended to support all prefixes.